lib/internal/modules/helpers.js

```diff
@@ -435,11 +443,54 @@ function stringify(body) {
   return DECODER.decode(body);
 }
 
+/**
+ * Enable on-disk compiled cache for all user modules being complied in the current Node.js instance
+ * after this method is called.
+ * If cacheDir is undefined, defaults to the NODE_MODULE_CACHE environment variable.
+ * If NODE_MODULE_CACHE isn't set, default to path.join(os.tmpdir(), 'node-compile-cache').
+ * @param {string|undefined} cacheDir
+ * @returns {{status: number, message?: string, directory?: string}}
+ */
+function enableCompileCache(cacheDir) {
+  if (cacheDir === undefined) {
+    cacheDir = join(tmpdir(), 'node-compile-cache');
+  }
+  const nativeResult = _enableCompileCache(cacheDir);
+  const result = { status: nativeResult[0] };
```

Comment:
```suggestion
  const result = { status: nativeResult[0], __proto__: null };
```

Reply:
I thought about that but this is going to be returned to the users, and we almost never set the prototype of objects returned to users. It's better to be consistent.